narratives/ncov_sit-rep_2020-05-01.md

```diff
@@ -0,0 +1,1743 @@
+<!-- This is left-side text 5-->
+# [Heterogeneous outbreak in Moscow](https://nextstrain.org/ncov/europe/2020-04-29?c=division&d=tree&f_country=Russia&f_division=Moscow&label=clade:A2&p=full&r=division&m=div)
+
+An example of frequent importation can be seen by zooming in on samples from Moscow.
+<br><br>
+There are a few clusters of cases which are most closely related to other samples from Moscow, indicating likely local transmission.
+However, most samples from Moscow are most closely related to samples from elsewhere, consistent with importation.
+<br><br>
+Given the number of confirmed cases in Moscow, it's likely that at least some of these introductions led to further local spread, but we don't yet have enough data to see this.
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 6-->
+# [Large outbreak cluster in St. Petersburg, sparked in early March](https://nextstrain.org/ncov/europe/2020-04-29?c=division&d=tree&f_country=Russia&f_division=Saint-Petersburg&f_region=Europe&label=clade:A2&p=full&r=division)
+
+We can look at St. Petersburg as a counterexample.
+<br><br>
+As with Moscow, there are samples from St. Petersburg across the tree, indicating that there have been multiple introductions.
+Contrastingly, if you click 'Explore the Data Yourself' and zoom into the main cluster in the center of this view, we can see a large clade that is made up entirely of samples from St. Petersburg.
+<br><br>
+This pattern is consistent with one of these introductions to St. Petersburg taking off to spark widespread transmission across the city, starting around the first week of March. This large outbreak cluster is most closely related to samples from Western Europe, although we cannot rule out the possibility of an unsampled, intermediate transmission from elsewhere.
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 7-->
+# [Five local transmission chains in the Czech Republic](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Czech%20Republic&m=div&p=full)
+
+The data from the Czech Republic show a mix of these two patterns.
+<br><br>
+There are at least 7 instances where a sample isolated in the Czech Republic descends from a virus that was mostly likely circulating elsewhere in Europe, indicating new introductions between March 1 and April 2.
+<br><br>
+For 5 of these introductions, we also see closely related local cases, sampled over time. This is consistent with local transmission.
+<br><br>
+From this, we can say that there were at least 7 introductions to the Czech Republic; at least 5 of these likely led to further local transmission.
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 8-->
+# [Extensive mixing of cases across Turkey](https://nextstrain.org/ncov/europe/2020-04-29?c=country_exposure&d=tree,map&f_country=Turkey&label=clade:A2&p=full&r=division&legend=open)
```

Comment:
```suggestion
# [Extensive mixing of cases across Turkey](https://nextstrain.org/ncov/europe/2020-04-29?c=country_exposure&d=tree,map&f_country=Turkey&label=clade:A2&p=full&r=division)
```
I'd have legends closed here myself - I found them taking up a lot of the screen - viewing on laptop sized screen.